python/cudf_polars/pyproject.toml

```diff
@@ -115,6 +114,9 @@ ignore = [
   "TD003", # Missing issue link on the line following this TODO
   # tryceratops
   "TRY003", # Avoid specifying long messages outside the exception class
+  # pyupgrade
+  "UP035",  # Import from `collections.abc` instead: `Callable`
+  "UP038",  # Use `X | Y` in `isinstance` call instead of `(X, Y)`
```

Comment:
@grlee77 pointed out to me that there's actually a performance penalty associated with using `|` instead of a tuple in `isinstance()`.

https://github.com/rapidsai/cucim/pull/766#issuecomment-2304727501